media-libs/netpbm/files/make-tarball.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
```

Comment:
rebase and drop this hunk as it's obsolete now

Reply:
Done